app/components/CopyAddressWarningModal.tsx

```
@@ -60,6 +60,19 @@ export const CopyAddressWarningModal: React.FC<
     imageUrl: token.imageUrl,
   }));
 
+  // Filter networks based on wallet type (same logic as NetworksDropdown):
+  // - If isInjectedWallet is true: show all networks (including Celo)
+  // - If isInjectedWallet is false: filter out Celo (smart wallet only)
+  const supportedNetworks = networks.filter(
```

Biome (2.1.2)

[error] Shouldn't redeclare 'supportedNetworks'. Consider to delete it or rename it.

'supportedNetworks' is defined here:


(lint/suspicious/noRedeclare)